lib/gollum/views/helpers.rb

```diff
@@ -43,6 +52,10 @@ def self.included(base)
           @@route_methods.to_json
         end
       end
+
+      def page_route(page)
+        "#{base_url}/#{page}".gsub(/\/{2,}/, '/')
```

Comment:
Could you add a comment for future reference to explain what substitution is being performed here?